developer_setup.md

```diff
@@ -155,9 +155,12 @@ git fetch other_user
 ### Get the Rails environment up and running
 
 ```bash
-bin/setup
+bin/setup                  # Installs dependencies, config, prepares database, etc
+bundle exec rake evm:start # Starts the ManageIQ EVM Application in the background
```

Comment:
do we need to run `evm:start` and rails s?

May want more of a note around configuring fewer resources / spartian stuff here.